packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -18,20 +18,40 @@ import {
 import sanitizeURL from '../shared/sanitizeURL';
 import {disableJavaScriptURLs} from 'shared/ReactFeatureFlags';
 
+import warning from '../shared/warnValidStyle';
```

Comment:
```suggestion
import warning from 'shared/warning';
```
could you try this instead?

Reply:
Thanks, I am trying

Reply:
```suggestion
import warning from '../shared/warnValidStyle';
import warning from 'shared/warning';
```
Try this

Reply:
```suggestion
import warning from 'shared/warning';
```